docs/rfcs/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+type: rfcs
```

Comment:
i added this for completeness sake and to make githubgen work correctly. owners list taken from current codeowners file